service/bookturks/quiz/QuizTools.py

```diff
@@ -22,24 +22,6 @@ def __init__(self):
         self.dbx = DropboxClient(settings.DROPBOX_CLIENT)
 
     @staticmethod
-    def get_quiz_id(username, quiz_name):
-        """
-        Concatenate to form the quiz_id
-        :param username:
-        :param quiz_name:
-        :return:
-        """
-        if not quiz_name.rstrip():
-            raise ValueError("Quiz Name cannot be blank")
-        quiz_id = "_".join([username, quiz_name.lower()])
-        if re.match("^[A-Za-z0-9_ -]*$", quiz_name):
```

Comment:
Yes it crashes quite a lot in PRs. Should get solved after merging.
Ill fix any broken tests.